edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Microsoft.Azure.Devices.Edge.Agent.Core.csproj

```diff
@@ -40,4 +41,12 @@
     <CodeAnalysisRuleSet>..\..\..\stylecop.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>  
   <Import Project="..\..\..\stylecop.props" />
+
+  <Target Name="ChangeAliasOfAkkaNet" BeforeTargets="FindReferenceAssembliesForReferences;ResolveReferences">
```

Comment:
why need this?

Reply:
Akka has some types which have the same name as the types we use elsewhere. So we need to quality the akka types with an alias.

Reply:
These should no longer be necessary since Akka.NET v1.4 - we were able to remove most of those type aliases now that we've moved onto .NET Standard 2.0.